### PR TITLE
Adds the i english element

### PR DESCRIPTION
**Pull request by kathyoleary**
:green_heart:  Last one of the elements and ready to go!  Need double thumbs please and thanks!

### CODE REVIEW

File: elements/i.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    </div>
+  </div>
+  <div class="how-to-remember">
+    <div class="how-to-remember-box how-to-remember-spanish">
+      <h4>&iquest;C&oacute;mo recordar?</h4>
+      <p></p>
+    </div>
+    <div class="how-to-remember-box how-to-remember-english">
+      <h4>How to Remember</h4>
+      <p>I is for interesting as well as italic and the &lt;i&gt; element is used to make interesting things stand apart by italicizing them.</p>
```
Comment (ericanaughton) [inline elements/i.html]: You may want to italicize the capital I at the beginning of this sentence, so that it stands out :)

Comment (kathyoleary) [inline elements/i.html]: Added it, I like that.  Great idea!

**Comment by werbeckes** [comment]
:+1:

**Comment by whazzmaster** [comment]
:+1:

**Pull request merged**